Sources/MapboxCoreNavigation/RoutesCoordinator.swift

```diff
@@ -11,21 +11,26 @@ final class RoutesCoordinator {
         case activeNavigation(UUID)
     }
 
-    typealias SetRoutesHandler = (RouteInterface?, _ legIndex: UInt32, _ completion: @escaping (Result<RouteInfo, Error>) -> Void) -> Void
+    typealias SetMainRouteHandler = (RouteInterface?, _ legIndex: UInt32, _ completion: @escaping (Result<RouteInfo, Error>) -> Void) -> Void
+    typealias SetAlternativeRoutesHandler = ([RouteInterface], _ completion: @escaping (Result<[RouteAlternative], Error>) -> Void) -> Void
```

Comment:
While we’re at it, we should give these type aliases less method-like names so that variables of this type can have less awkward names. How about `MainRouteSelectionHandler` and `AlternativeRoutesAvailabilityHandler`?

Reply:
I feel it's better to have both main and alternative handlers to share similar naming pattern because they are doing similar action.